Makefile

```diff
@@ -18,14 +18,12 @@ gaia:
 	go build $(BUILD_FLAGS) -o build/gaiacli ./examples/gaia/gaiacli
 
 build:
-	@rm -rf $(shell pwd)/examples/basecoin/vendor/
-ifeq ($(OS),Windows_NT)
```

Comment:
oh, I guess the conditional was necessary for the `.exe` ?

Reply:
Oh - we want the conditional for build but not for install. I'll fix this and push to this branch and merge, thanks